packages/lodestar-beacon-state-transition/src/fast/index.ts

```diff
@@ -10,10 +10,9 @@ export {IStateContext, EpochContext};
 export function fastStateTransition(
   {state, epochCtx}: IStateContext,
   signedBlock: SignedBeaconBlock,
-  verifyStateRoot = true,
-  verifyProposer = true,
-  verifySignatures = true
+  options?: {verifyStateRoot?: boolean; verifyProposer?: boolean; verifySignatures?: boolean}
 ): IStateContext {
+  const {verifyStateRoot = false, verifyProposer = true, verifySignatures = true} = options || {};
```

Comment:
I think we should have everything to true by default

Reply:
I respected the original values and logic. Should I change it anyway?

Reply:
DO we every use default values anywhere?

Reply:
In one test in packages/lodestar

Reply:
I think it's fine to change defaults then and just update there